src/components/Feed.tsx

```diff
@@ -5,15 +5,15 @@ import { Button } from "antd";
 import { useAppSelector } from "../redux/hooks";
 
 enum FeedTypes {
-  FEED,
+  MY_FEED,
   MY_POSTS,
-  ALL_POSTS,
+  DISCOVER,
 }
 
 const Feed = (): JSX.Element => {
   const userId = useAppSelector((state) => state.user.id);
   const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
-  const [feedType, setFeedType] = useState<FeedTypes>(FeedTypes.FEED);
+  const [feedType, setFeedType] = useState<FeedTypes>(FeedTypes.DISCOVER);
 
   const feedNavClassName = (navItemType: number) =>
```

Comment:
Not something you changed, but this should also take the enum
```suggestion
  const feedNavClassName = (navItemType: FeedTypes) =>
```